pages/about/student-opportunities/index.vue

```diff
@@ -0,0 +1,212 @@
+<script setup>
+// HELPERS
+import _get from 'lodash/get'
+import fixUri from '../utils/fixUri'
+import removeTags from '../utils/removeTags'
+
+// GQL
+import STUDENT_OPPORTUNITIES_LIST from '../gql/queries/JobStudentOpportunitiesList.gql'
+
+const { $graphql } = useNuxtApp()
+
+const { data } = await useAsyncData('student-opportunities-list', async () => {
```

Comment:
Throw error when useasyncdata fails and sends an  error object check and also throw error if data is empty like the slug pages

Reply:
Add a watcher for  data refs when preview mode is enable check previewmode PR https://github.com/UCLALibrary/library-website-nuxt/pull/787